panel/widgets/select.py

```diff
@@ -1361,12 +1359,12 @@ def _apply_selection(self, event):
         """
         selected = event.obj is self._buttons[True]
 
-        new = OrderedDict([(k, self._items[k]) for k in self._selections[not selected]])
+        new = {(k, self._items[k]) for k in self._selections[not selected]}
```

Comment:
This is wrong (and the same below). You are making a set with tuples, not a dictionary.